src/global/global_init.cc

```diff
@@ -154,11 +154,6 @@ void global_pre_init(
   // command line (as passed by caller)
   conf.parse_argv(args);
 
-  if (conf->log_early &&
```

Comment:
I am not sure how this matches up with the commit description.

Reply:
@neha-ojha 
https://github.com/ceph/ceph/blob/master/src/global/global_init.cc#L177 shows that log_early condition makes non-sense here.

Reply:
`log_early` option was introduced in #27419 so we could see the logs when talking to monitor in thebootstrap phase, back then, we connected to monitor in between these two `cct->_log->start()` calls. and later, in #28012, we connect the monitor in `global_init()` after `setuid()` gets called.

@batrick how about removing this option / argument and always starting logger at https://github.com/ceph/ceph/pull/27419/files#diff-36404ac17f11600f7f803ca7d716a73dR84 ?

Reply:
Makes sense to me. @changchengx please make that change.

Reply:
@tchaikov librados needs the same change too right?

Reply:
yeah. i missed `src/librados/RadosClient.cc` actually. @changchengx could you help update librados accordingly?

Reply:
@tchaikov Done